content/kubermatic/v2.17/tutorials_howtos/upgrading/upgrade_from_2.16_to_2.17/kubermatic_operator/_index.en.md

```diff
@@ -0,0 +1,89 @@
++++
+title = "Upgrading KKP Operator"
+date = 2020-06-09T11:09:15+02:00
+weight = 10
+
++++
+
+Upgrading a KKP setup that is already managed by the Operator is as simple as
+updating the Helm charts and following the [general upgrade notes]({{< ref "../" >}}).
+
+## Upgrade Procedure
+
+Download the [latest 2.17 release](https://github.com/kubermatic/kubermatic/releases) from GitHub
+(make sure to choose the right version, CE for the Community or EE for the Enterprise Edition) and
+extract the archive locally.
+
+```bash
+wget https://github.com/kubermatic/kubermatic/releases/download/v2.17.0/kubermatic-ee-v2.17.0-linux-amd64.tar.gz
+tar -xzvf kubermatic-ce-v2.17.0-linux-amd64.tar.gz
+```
+
+Run the installer to perform the upgrade using the `kubermatic.yaml` and `values.yaml` you used to create the platform.
+
+```bash
+./kubermatic-installer deploy --config kubermatic.yaml --helm-values values.yaml --migrate-cert-manager
+```
+
+Note that the `--migrate-cert-manager` flag is necessary for the migration to cert-manager `1.2.0` to succeed.
+It will trigger an automatic migration of all `Certificate` resources from
+version `v1alpha1` to `v1` (find more details [here](https://github.com/kubermatic/kubermatic/pull/6739)).
+
+
+The Operator will automatically update the KKP Seed Controller Manager on every seed cluster.
+Manually upgrade all other charts you might have installed as part of the monitoring or logging
+stacks.
+
+## Migrating from the `nodeport-proxy` Helm Chart
```

Comment:
This seems unnecessary as we have already convered it in `Migrating to Operator`.
WDYT?

Reply:
This is just because it is possible that some users already migrated to the operator but still have np-proxy deployed with the helm chart. Anyway as @shaase-ctrl pointed out duplication is not a sin in scope of documentation ;-)